BlockServer/epics/archiver_manager.py

```diff
@@ -93,4 +96,13 @@ def _generate_archive_channel(self, group, block_prefix, block):
             else:
                 period.text = str(datetime.timedelta(seconds=1))
                 monitor = eTree.SubElement(channel, 'monitor')
-                monitor.text = str(block.log_deadband)
\ No newline at end of file
+                monitor.text = str(block.log_deadband)
+        else:
+            # Blocks that aren't logged, but are needed for the dataweb view
+            channel = eTree.SubElement(dataweb, 'channel')
+            name = eTree.SubElement(channel, 'name')
+            name.text = block_prefix + block.name
+            period = eTree.SubElement(channel, 'period')
+            period.text = str(datetime.timedelta(seconds=5))
```

Comment:
Hi Kathryn, does using

           period.text = str(datetime.timedelta(seconds=300))
           monitor = eTree.SubElement(channel, 'scan')

for non-logged work? The docs are a little unclear if current values will get updated all the time with archiving happening only every 300 seconds, or whether it will poll every 300 seconds. If the former, it would help avoid unnecessary logging.